### ISSUE TITLE
NTR: Neuroleptic sensitivity (Triggers or responses: drugs or treatments)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9473]
New term request Neuroleptic sensitivity (kalia_091020003812): Severe cognitive and motor adverse reaction to antipsychotic agents; thioridazine, chlorpromazine, thiothixene

**Comment by pnrobinson** [comment]
@MickeySegal
I think this would be a good addition to the [Triggered by exposure to medication HP:4000101](https://hpo.jax.org/app/browse/term/HP:4000101)  subontology and would label and define this term qccordingly if you agree?

**Comment by MickeySegal** [comment]
Agree.

**Comment by pnrobinson** [comment]
@MickeySegal 
added as
HP:0034551, Triggered by neuroleptic medication
